tests/virt_autotest/reboot_and_wait_up.pm

```diff
@@ -59,8 +59,8 @@ sub reboot_and_wait_up {
                 #Xen console may output additional messages about vm on sol whose output is disrupted.
                 #So in order to get login prompt back on screen, 'ret' key should be fired up. But the
                 #os name banner might not be available anymore, only 'linux-login' needle can be matched.
+                send_key 'ret' for (0 .. 2);
```

Comment:
It seems, with this change, kvm and xen branches below will have the same logic. Better to merge them into one/not differentiate?

Reply:
Done.